### ISSUE TITLE
Keep older versions on new releases

### ISSUE THREAD

**Issue by joe4dev** [bordoni/fakerpress#51]
Could you keep older versions when you release a new version on https://wordpress.org/plugins/fakerpress/ ?
Automatic plugin installation via wp-cli (http://wp-cli.org/) will fail otherwise (e.g., `wp plugin install fakerpress --version=0.2.2` attempts to install version 0.2.2 which is not available anymore)

**Comment by bordoni** [comment]
Yes, that is a problem that I need to fix for sure. We do have them tagged here on GitHub.

https://github.com/bordoni/fakerpress/tags

**Comment by joe4dev** [comment]
Great, meanwhile `wp plugin install https://github.com/bordoni/fakerpress/archive/0.2.2.zip` would be a workaround according to http://wp-cli.org/commands/plugin/install/

**Comment by bordoni** [comment]
Just one question, why would rather have an older version of the plugin? Something I've changed?

**Comment by joe4dev** [comment]
I'm working on a [Wordpress benchmark](https://github.com/sealuzh/cwb-benchmarks/tree/master/wordpress-bench) for our research tool [Cloud WorkBench](https://github.com/sealuzh/cloud-workbench/) and want provide reproducibility (lock version).
I've written a [Ruby client](https://github.com/sealuzh/cwb-benchmarks/blob/master/wordpress-bench/libraries/faker_press.rb) therefore which I've adapted for 0.3.x but it might break on future versions.

**Comment by bordoni** [comment]
Very cool solution @joe4dev! In the future I intend to create a `wp-cli` Add-on to FakerPress, to allow this kind of benchmarking tools to be faster.

**Comment by bordoni** [comment]
Moved this out of the release, since it's not a code thing. It's more of adding the zips there.

**Comment by bordoni** [comment]
Resolved this on version 0.8.0
